src/rpc/net.cpp

```diff
@@ -147,6 +151,18 @@ static UniValue getpeerinfo(const JSONRPCRequest& request)
         if (stats.addrBind.IsValid())
             obj.pushKV("addrbind", stats.addrBind.ToString());
         obj.pushKV("services", strprintf("%016x", stats.nServices));
+        UniValue servicesNames(UniValue::VARR);
+        if (stats.nServices & NODE_NETWORK)
+            servicesNames.push_back("NETWORK");
+        if (stats.nServices & NODE_GETUTXO)
+            servicesNames.push_back("GETUTXO");
+        if (stats.nServices & NODE_BLOOM)
+            servicesNames.push_back("BLOOM");
+        if (stats.nServices & NODE_WITNESS)
+            servicesNames.push_back("WITNESS");
+        if (stats.nServices & NODE_NETWORK_LIMITED)
+            servicesNames.push_back("NETWORK_LIMITED");
+        obj.pushKV("servicesnames", servicesNames);
```

Comment:
Couldn't this be in a function, so that only one place has to be updated in the future?
```suggestion
        obj.pushKV("servicesnames", GetServicesNames(stats.nServices));
```

Reply:
Fixed